package.json

```diff
@@ -8,7 +8,7 @@
     "node-sass": "^4.13.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.2.0"
+    "react-scripts": "^3.2.0"
```

Comment:
If this change wasn't intentional, revert both `package.json` and `package-lock.json` to their original states. 😊

Reply:
I am sorry I forgot to look after this message @praveenscience Sir, I thought I'll get the notifications about the message replies. But I wasn't getting them. So sorry. 
And about the issue you requested for changes wasn't intentional they might have happened during NPM install.

Reply:
No worries. Can you revert these two files back as the original?